### PR TITLE
[Merged by Bors] - doc: separable closure of F in E

### PR DESCRIPTION
**Pull request by alreadydone**
Change the expression "separable closure of `E / F`" to "separable closure of `F` in `E`", which IMO is closer to everyday usage. I think we should do the same for "perfect closure" in #9488.

Also remove the use of some parenthesized adjectives: IMO "(some adjective)" signifies that "some adjective" will be assumed to be the default in subsequent text, making it unnecessary to mention "some adjective" every time. This applies to "(relative)" and "(infinite)" in the file SeparableClosure. Writing both "(infinite)" and "(finite)" signifies that both are assumed to be default, which is not our intention (I think we intend to make "infinite" the default). Therefore I kept the first "(infinite)" and removed all subsequent "(infinite)" and parentheses around "finite".

In the file SeparableDegree, the infinite separable degree is not yet defined, so "finite" is the default instead: indeed the file omits "finite" almost everywhere, so I just remove the remaining three occurrences of "(finite)".

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/SeparableDegree.lean [hunk 1]
```diff
@@ -245,7 +245,7 @@ def embProdEmbOfIsAlgebraic [Algebra E K] [IsScalarTower F E K] (halg : Algebra.
           halg).restrictScalars F).symm
 
 /-- If `K / E / F` is a field extension tower, such that `K / E` is algebraic, then their
-(finite) separable degree satisfies the tower law
+separable degrees satisfy the tower law
 $[E:F]_s [K:E]_s = [K:F]_s$. See also `FiniteDimensional.finrank_mul_finrank'`. -/
```
Comment (alreadydone) [inline Mathlib/FieldTheory/SeparableDegree.lean:249]:
```suggestion
$[E:F]_s [K:E]_s = [K:F]_s$. See also `FiniteDimensional.finrank_mul_finrank`. -/
```

File: Mathlib/FieldTheory/SeparableDegree.lean [hunk 2]
```diff
@@ -37,11 +37,11 @@ This file contains basics about the separable degree of a field extension.
   closure of `E`, as a natural number. It is zero if `Field.Emb F E` is not finite.
   Note that if `E / F` is not algebraic, then this definition makes no mathematical sense.
 
-  **Remark:** the (infinite) separable degree `Field.sepDegree F E` for a general algebraic
-  extension `E / F` is defined to be the degree of `L / F`, where `L` is the (relative) separable
-  closure `separableClosure F E` of `E / F`, which is not defined in this file yet. Later we
-  will show that (`Field.finSepDegree_eq`), if `Field.Emb F E` is finite, then these two
-  definitions coincide.
+  **Remark:** the `Cardinal`-valued, potentially infinite separable degree `Field.sepDegree F E`
+  for a general algebraicextension `E / F` is defined to be the degree of `L / F`, where `L` is
```
Comment (acmepjz) [inline Mathlib/FieldTheory/SeparableDegree.lean:41]:
```suggestion
  for a general algebraic extension `E / F` is defined to be the degree of `L / F`, where `L` is
```

**Review by acmepjz — COMMENTED** [review]
Thanks for the docs correction, it looks good to me. I'll modify the corresponding docs in my PR.

**Comment by acmepjz** [comment]
Is this PR ready to merge?

File: Mathlib/FieldTheory/SeparableClosure.lean [hunk 3]
```diff
@@ -284,12 +283,12 @@ theorem sepDegree_eq_of_equiv (K : Type v) [Field K] [Algebra F K] (i : E ≃ₐ
     sepDegree F E = sepDegree F K :=
   i.separableClosure.toLinearEquiv.rank_eq
 
-/-- The (infinite) separable degree multiply by the (infinite) inseparable degree is equal
+/-- The separable degree multiply by the inseparable degree is equal
```
Comment (jcommelin) [inline Mathlib/FieldTheory/SeparableClosure.lean:286]:
```suggestion
/-- The separable degree multiplied by the inseparable degree is equal
```

File: Mathlib/FieldTheory/SeparableDegree.lean [hunk 4]
```diff
@@ -37,11 +37,11 @@ This file contains basics about the separable degree of a field extension.
...
   Note that if `E / F` is not algebraic, then this definition makes no mathematical sense.
 
-  **Remark:** the (infinite) separable degree `Field.sepDegree F E` for a general algebraic
-  extension `E / F` is defined to be the degree of `L / F`, where `L` is the (relative) separable
-  closure `separableClosure F E` of `E / F`, which is not defined in this file yet. Later we
-  will show that (`Field.finSepDegree_eq`), if `Field.Emb F E` is finite, then these two
-  definitions coincide.
+  **Remark:** the `Cardinal`-valued, potentially infinite separable degree `Field.sepDegree F E`
+  for a general algebraic extension `E / F` is defined to be the degree of `L / F`, where `L` is
+  the (relative) separable closure `separableClosure F E` of `E / F`, which is not defined in
```
Comment (jcommelin) [inline Mathlib/FieldTheory/SeparableDegree.lean:42]:
```suggestion
  the (relative) separable closure `separableClosure F E` of `F` in `E`, which is not defined in
```

**Review by jcommelin — APPROVED** [review]
bors d+

**Comment by alreadydone** [comment]
Thanks!
bors r+

**Pull request closed without merging**